cilium/bpf_metadata.cc

```diff
@@ -395,10 +389,11 @@ Config::extractSocketMetadata(Network::ConnectionSocket& socket) {
 
   // Load the policy for the Pod that sends or receives traffic.
   // Might change later on for North/South L7LB traffic.
-  auto policy = getPolicy(pod_ip);
+  // Use pointer so that we can change 'policy' below if needed.
```

Comment:
nit: -> `below (for "North/South L7 LB" use-case) if needed`

Reply:
Changed to:
```
  // Use a pointer as we may need to change the policy in the case of "North/South L7 LB" below.
```